buildSrc/src/main/kotlin/Constants.kt

```diff
@@ -221,7 +221,7 @@ object Prometheus {
     }
 }
 
-const val RapidAndRivers = "com.github.navikt:rapids-and-rivers:20210428115805-514c80c"
```

Comment:
Rapid og rivers har bytta versjoneringsregime, med denne endringen går vi tilbake. ref https://github.com/navikt/rapids-and-rivers/commit/e31cd735a509760b0bda7acd8ef9c945185efb0f og https://github.com/navikt/rapids-and-rivers/ jitpack badge

Reply:
jepp fuck opp commiten